build/config.props

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0">
+
+  <!-- Special project import for dotnet source build
```

Comment:
I moved this because it always has to be at the absolute top of the file to avoid accidental overrides.